src/CategoriesContainer.jsx

```diff
@@ -19,9 +22,12 @@ export default function CategoriesContainer() {
   }
 
   return (
-    <ul>
+    <MenuList>
       {categories.map((category) => (
-        <li key={category.id}>
+        <MenuItem
+          key={category.id}
+          active={selectedCategory && category.id === selectedCategory.id}
```

Comment:
category의 id가 undefined가 아니라는 가정하에 간단하게 `optional chaining`을 이용해서 아래처럼 표현해줄 수도 있을 것 같네요
```suggestion
          active={selectedCategory?.id === category.id}
```

Reply:
`optional chaining` 표현에 대해 배웠습니다. 감사합니다 >____<